### PR TITLE
feature/17425 - Sticky banner dynamic positioning.

### PR DESCRIPTION
**Pull request by pris54**
With this modification the sticky banner remains always above the footer. 
Tested on Chrome, Firefox and Safari for Mac. IE 9, 10, 11. Chrome, Safari and Firefox for Windows. iPad(s).

### CODE REVIEW

File: admin/assets/javascripts/global-scripts.js [hunk 1]
```diff
@@ -154,6 +153,7 @@ $(function () {
      * @param status
      */
     toggle: function (status) {
+      stickyBannerPositioning()
```
Comment (GeorgeChatzigiannis) [inline admin/assets/javascripts/global-scripts.js]: was thinking alternative naming for this convention.
how about adjustStickyBanner?
From what I noticed in this file the general approach is verb first.

**Comment by activemq** [comment]
Tested across different browsers, a few issues with mobiles, however paula has agreed with sinead for these not to be a priority for the trial.

File: admin/assets/javascripts/global-scripts.js (hunk 1 above)
Comment (jon-shipley) [reply admin/assets/javascripts/global-scripts.js]: I think noun-verb naming is correct

File: admin/assets/javascripts/global-scripts.js [hunk 2]
```diff
@@ -253,6 +253,26 @@ $(function () {
     }
   }
 
+  /**
+   * Sticky banner positioning.
+   */
+  function stickyBannerPositioning () {
+    var windowHeight = $(window).height()
```
Comment (jon-shipley) [inline admin/assets/javascripts/global-scripts.js]: The admin app should be functional without javascript.  What happens here?

Comment (pris54) [reply admin/assets/javascripts/global-scripts.js]: @jon-shipley We are going for JS enabled only, I've confirmed this three times.

File: admin/assets/javascripts/global-scripts.js (hunk 1 above)
Comment (pris54) [reply admin/assets/javascripts/global-scripts.js]: @jon-shipley @GeorgeChatzigiannis happy if I rename it adjustStickyBanner? Please confirm.

File: admin/assets/javascripts/global-scripts.js (hunk 2 above)
Comment (jon-shipley) [reply admin/assets/javascripts/global-scripts.js]: I see. That's fine then.

**Pull request merged**